File: load-places.py
import json
import urllib2

for i in xrange(100000):
    payload = {
        'name': "Python Place %d" %i,
        'streetAddress':  "%d Street Name" %i,
        'city': "%d City" %(i % 1000),
        'state': "%d state" %(i % 100),
        'zip': "12345" }

    print("payload: ", json.dumps(payload))

    req = urllib2.Request('http://localhost:8080/places')
    req.add_header('Content-Type', 'application/json')

    response = urllib2.urlopen(req, json.dumps(payload))
    print("response code: %d" %response.code)
    response.close()

File: load-places.rb
require "net/http"
require "uri"
require "json"

uri = URI.parse("http://localhost:8080/places")

(1..200000).each do |i|
    req = Net::HTTP::Post.new(uri, initheader = { 'Content-Type' => 'application/json' })
    payload = { name: "Place #{i}",
        streetAddress:  "#{i} Street Name",
        city: "#{i % 1000} City",
        state: "#{i % 100} state",
        zip: "12345" }
    req.body = payload.to_json
    res = Net::HTTP.start(uri.hostname, uri.port) do |http|
        http.request(req)
    end
    puts "response: #{res.body}"
end